cohortextractor/patients.py

```diff
@@ -565,6 +566,9 @@ def with_these_clinical_events(
             ignored. if a events is found on this day, the date is not matched even it matches a
             code in the main `codelist`
         episode_defined_as: a string expression indicating how an episode should be defined
+        numeric_value_in_range: a pair of numbers `(lower, upper)`.  Only events whose
+            numeric value is between `lower` and `upper` are matched.  Either `lower` or
+            `upper` may be `None`.
```

Comment:
Should we include whether they're inclusive/exclusive explicity here?  It feels like something would be useful to call out?